ang/civicase/contact/services/contacts-cache.service.js

```diff
@@ -111,5 +110,23 @@
     this.getContactIconOf = function (contactID) {
       return savedContactDetails[contactID] ? savedContactDetails[contactID].contact_type : '';
     };
+
+    /**
+     * Formats the phone numbers in an array which can be used in the html
```

Comment:
Add `.` in the comment end.

Reply:
As there is already a linter in place, and the purpose of the linter is to eliminate any personal preferences related to how to format the code/comments. So we should not care about formatting codes anymore, unless the linter complaints.
Only exception to this, at the moment, would be, if we forget about documenting the parameters, which at the moment semistandard does not complain about. I will see if we can add that as part of the configuration of linter. 
But overall, we should care about anything related to code formatting, unless the linter complaints.